src/msw/listctrl.cpp

```diff
@@ -3569,4 +3570,14 @@ static void wxConvertToMSWListCol(HWND hwndList,
     }
 }
 
+#if wxUSE_TOOLTIPS
+
+void wxListCtrl::DoSetToolTip( wxToolTip *tip )
+{
+    MSWSetExListStyles( tip );
+	wxWindow::DoSetToolTip( tip );
```

Comment:
Do _**you**_ ever _**review**_ your changes?
At the beginning of every PR, at the top of the github page there are a bunch of tabs: `Conversation`, `Commits`, `Checks`, **`Files changed`**. Go to the last one **every time** you have changes, and scroll down to **review** them! It will be very easy to spot at least formatting issues.